src/Security/UserClassBuilder.php

```diff
@@ -259,4 +261,17 @@ private function addEraseCredentials(ClassSourceManipulator $manipulator): void
 
         $manipulator->addMethodBuilder($builder);
     }
+
+    private function addUniqueConstraint(ClassSourceManipulator $manipulator, UserClassConfiguration $userClassConfig): void
+    {
+        if ($userClassConfig->isEntity()) {
```

Comment:
Let's exit early when possible.
```suggestion
        if (!$userClassConfig->isEntity()) {
            return;
        }
        
```